updates/commands.go

```diff
@@ -119,6 +123,111 @@ func createStatusMessage(user users.User, chatId int64, msg tgbotapi.MessageConf
 	return msg
 }
 
+func handleMonthlySummaryCommand(fatBotUpdate FatBotUpdate) {
+	update := fatBotUpdate.Update
+	bot := fatBotUpdate.Bot
+	originalChatId := update.FromChat().ID
+	user, err := users.GetUserFromMessage(update.Message)
+	if err != nil {
+		log.Error(err)
+		sentry.CaptureException(err)
+		return
+	}
+
+	if user.ID == 0 {
+		return
+	}
+	months := 6 // until we'll let the user enter the number of months
+	start, end := getDuration(months)
+	chatIds, err := user.GetChatIds()
+	if err != nil {
+		log.Error(err)
+		sentry.CaptureException(err)
+		return
+	}
+	if len(chatIds) > 0 {
+		for _, chatId := range chatIds {
+			createUserMonthlySummaryChart(bot, originalChatId, user, chatId, start, end)
+		}
+	} else {
+		log.Errorf("No chat IDs found for user %s", user.Username)
+	}
+}
+func createUserMonthlySummaryChart(bot *tgbotapi.BotAPI, originalChatId int64, user users.User, chatId int64, start, end time.Time) {
+	group, _ := users.GetGroup(chatId)
+	fileName := fmt.Sprintf("%s_%s.png", user.Username, group.Title)
+	months := getMonthsInRange(start, end)
+	userWorkoutsByMonthAsString := user.GetMonthlyWorkoutsSummary(group, start, end, months)
+	monthsStringSlice := "'" + strings.Join(months, "', '") + "'"
+	userWorkoutsByMonthStringSlice := strings.Join(userWorkoutsByMonthAsString, ", ")
+	chartConfig := createMonthlyChartConfig(monthsStringSlice, userWorkoutsByMonthStringSlice)
+	qc := createQuickChart(chartConfig)
+	file, err := os.Create(fileName)
+	if err != nil {
+		panic(err)
+	}
+	defer file.Close()
+	qc.Write(file)
+	msg := tgbotapi.NewPhoto(originalChatId, tgbotapi.FilePath(fileName))
+	msg.Caption = fmt.Sprintf("Monthly Workouts summary\n")
+
+	if _, err = bot.Send(msg); err != nil {
```

Comment:
Instead of sending the message here, return it to the place above where all messages are sent centrally (where you configured the new `summary` command inside the switch)